pkg/network/protocols/http/monitor.go

```diff
@@ -101,6 +105,17 @@ func (m *Monitor) Start() error {
 	return m.processMonitor.Initialize()
 }
 
+func (m *Monitor) GetUSMStats() map[string]interface{} {
+	if m == nil {
+		return map[string]interface{}{
+			"Error": USMStartupError.Error(),
+		}
+	}
+	return map[string]interface{}{
+		"last_check": m.telemetry.then,
```

Comment:
When is this timestamp updated? Do we intend to update it every time `GetUSMStats` is called?

Reply:
every time `GetHTTPStats` is being called, it triggers `m.telemetry.log()` which updates the time